Four tests failed on win-64:

```
            expected = "Error in file %s" % os.path.join(config_path, 'test_error.ini')

>           assert str(e).split(':')[0] == expected

E           AssertionError: assert 'Error in file C' == 'Error in fil...est_error.ini'

E

E             - Error in file %SRC_DIR%\tests\sample_configs\test_error.ini

E             + Error in file C
=========================== short test summary info ===========================

FAILED tests/test_config.py::test_config1 - AssertionError: assert {'apppath'...

FAILED tests/test_config.py::test_config2 - AssertionError: assert {'apppath'...

FAILED tests/test_config.py::test_config3 - AssertionError: assert {'apppath'...

FAILED tests/test_config.py::test_interpolate_exception - AssertionError: ass...
=================== 4 failed, 22 passed, 1 skipped in 0.60s ===================
```

There is something bad with files (missing test files in our test environment?) or paths on Windows specifically. Check if we can fix it or if there are the upstream issues https://github.com/Pylons/pastedeploy/issues?q=sort%3Aupdated-desc%20is%3Aissue

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -40,8 +40,9 @@ test:
   commands:
     - sed -i "s/\//\\\\/g" tests\sample_configs\test_config.ini # [win]
     - pip check
-    - pytest -v tests -k "not test_interpolate_exception" # [win]
     - pytest -v tests # [not win]
+    # Skip due to issue with paths on Windows (affects only test)
+    - pytest -v tests -k "not test_interpolate_exception" # [win]
 
 about:
   home: https://docs.pylonsproject.org/projects/pastedeploy

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,7 +36,9 @@ test:
   requires:
     - pip
     - pytest
+    - msys2-sed  # [win]
   commands:
+    - sed -i "s/\//\\\\/g" tests\sample_configs\test_config.ini # [win]
     - pip check
     - pytest -v tests
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -40,7 +40,8 @@ test:
   commands:
     - sed -i "s/\//\\\\/g" tests\sample_configs\test_config.ini # [win]
     - pip check
-    - pytest -v tests
+    - pytest -v tests -k "not test_interpolate_exception" # [win]
+    - pytest -v tests # [not win]
 
 about:
   home: https://docs.pylonsproject.org/projects/pastedeploy

